DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: s3mpi
 Title: R message passing interface using S3 storage
 Description: Easily pass objects like lists or dataframes between consoles
-Version: 0.2.2
+Version: 0.2.3
```

Comment:
Good questions. I agree that semver is the way to go to be consistent with the rest of the ecosystem. If the bug is not major I think Hadley does not put it in `NEWS.md` either.